### PR TITLE
Added turning flowchart; wind direction conventions

### LINKED ISSUE #82: Update docs page Reference/Sailing Knowledge
### Changes
- Page Tree name of types of turns page should be "Types of Turn**s**"
- Create flow chart for types of turns
- Talk about wind convention

### Resources
- Flow chart
    - https://squidfunk.github.io/mkdocs-material/reference/diagrams/#usage
    - https://ubcsailbot.github.io/docs/reference/github/workflow/#development-workflow-overview

### PR DESCRIPTION
**Pull request by jamenkaye**
This PR will resolve #82 when it is completed.

- What do I need to do with the page tree? It seems like "Types of Turn" is already capitalized in the page tree in mkdocs.yml
- I created the flow chart. Do you know if there's a way to split the contents of a box over multiple lines? I couldn't find a way to do this so I used grammatically incomplete questions so that the font was legible and the boxes weren't too large.
- I put the wind convention in the miscellaneous page

### CODE REVIEW

**Comment by patrick-5546** [comment]
- Just add an "s": "Types of Turn" -> "Type of Turns" to match the page header
- @DFriend01 can you help with the flowchart stuff
- Wind conventions section looks good

**Review by patrick-5546 — COMMENTED** [review]
- Just add an "s": "Types of Turn" -> "Type of Turns" to match the page header
- @DFriend01 can you help with the flowchart stuff
- Wind conventions section looks good

**Comment by jamenkaye** [comment]
- Ok, the s is added haha
- 👍
- thanks!

**Comment by DFriend01** [comment]
@jamenkaye You have to use the HTML tag `<br/>` to break onto a newline. Here is an example of me using it on your flowchart:

# Without `<br/>`

```
graph LR
    B[Classify a Turn] --> C{Sail change sides?};
    C --> |Yes| E{"Which side of the boat is the upwind end?"};
    C --> |No| D{Pointing higher or lower?};
    D --> |Higher| F[Heading Up];
    D --> |Lower| G[Bearing Off];
    E --> |Bow| H[Tack];
    E --> |Stern| I[Gybe];
```

```mermaid
graph LR
    B[Classify a Turn] --> C{Sail change sides?};
    C --> |Yes| E{"Which side of the boat is the upwind end?"};
    C --> |No| D{Pointing higher or lower?};
    D --> |Higher| F[Heading Up];
    D --> |Lower| G[Bearing Off];
    E --> |Bow| H[Tack];
    E --> |Stern| I[Gybe];
```

# With `<br/>`

```
graph LR
    B[Classify a Turn] --> C{Sail change sides?};
    C --> |Yes| E{"Which side of the boat <br/>is the upwind end?"};
    C --> |No| D{Pointing higher or lower?};
    D --> |Higher| F[Heading Up];
    D --> |Lower| G[Bearing Off];
    E --> |Bow| H[Tack];
    E --> |Stern| I[Gybe];
```

```mermaid
graph LR
    B[Classify a Turn] --> C{Sail change sides?};
    C --> |Yes| E{"Which side of the boat <br/>is the upwind end?"};
    C --> |No| D{Pointing higher or lower?};
    D --> |Higher| F[Heading Up];
    D --> |Lower| G[Bearing Off];
    E --> |Bow| H[Tack];
    E --> |Stern| I[Gybe];
```

**Comment by jamenkaye** [comment]
Thanks @DFriend01! I think it looks much better now.

@patrick-5546 now I've _actually_ completed all the changes I wanted to make for the linked issue #82. Sorry I probably should have waited until now to mark it as "Ready for Review". Let me know if you have any feedback :)

File: docs/reference/sailing/turning.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
 # Types of Turns
 
-In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagram.
+In sailing, there are 4 distinct types of turns. Read the descriptions below, and observe how they fit into the diagrams.
 
 Note that any of these types of turn can be done in either the clockwise or counter-clockwise directions.
 
+## Classifying Types Of Turns Summary
+
```
Comment (patrick-5546) [inline docs/reference/sailing/turning.md:8]: Could put a sentence here to describe the flowchart

File: docs/reference/sailing/turning.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+``` mermaid
+graph LR
+    B[Classify a Turn] --> C{Does the sail change<br/>sides during the turn?};
+    C --> |Yes| E{Which end of<br/>the boat is upwind<br/>during the turn?};
+    C --> |No| D{Does the<br/>boat point higher<br/>or lower at the end<br/>of the turn?};
```
Comment (patrick-5546) [inline docs/reference/sailing/turning.md:18]: Might be good to specify higher or lower respective to what. Higher is more upwind right?

**Review by patrick-5546 — CHANGES_REQUESTED** [review]

File: docs/reference/sailing/turning.md (hunk 1 above)
Comment (jamenkaye) [reply docs/reference/sailing/turning.md:8]: I added this. I was kinda hoping the title would speak for itself haha but I don't mind adding a sentence.

File: docs/reference/sailing/turning.md (hunk 2 above)
Comment (jamenkaye) [reply docs/reference/sailing/turning.md:18]: That's correct, pointing higher is pointing closer towards straight into the wind. I was sorta hoping people would read the rest of the page to see where I defined "higher" and "lower" but its probably better that I move those definitions up here instead haha

**Review by patrick-5546 — APPROVED** [review]
Looks good to me, feel free to merge

**Pull request merged**